exchange/css.go

```diff
@@ -425,13 +418,15 @@ func GetObjectDataByChunk(ec ExchangeContext, org string, objType string, objId
 				return false, fmt.Errorf("Failed to seek to the offset from beginning of a file. Error: %v", err)
 			}
 
-			if _, err := io.Copy(file, response.Body); err != nil && err != io.EOF {
+			if written, err := io.Copy(file, response.Body); err != nil && err != io.EOF {
 				return false, fmt.Errorf("Failed to write to file. Error: %v", err)
+			} else if written != int64(endOffset-startOffset+1) {
+				return false, fmt.Errorf("Failed to write all the data to file.")
 			}
 
 			return true, nil
 		} else {
-			return false, fmt.Errorf("Failed to get chunk data of object %s/%s", objType, objId)
+			return false, fmt.Errorf("Failed to get chunk data of object %s/%s. Response code was %v.", objType, objId, response.StatusCode)
```

Comment:
if response is nil, the `response.StatusCode` would throw out nil pointer

Reply:
Fixed